plugins/automation/lib/discourse_automation/event_handlers.rb

```diff
@@ -206,21 +206,23 @@ def self.handle_topic_tags_changed(topic, old_tag_names, new_tag_names)
             next if !watching_categories["value"].include?(topic.category_id)
           end
 
+          missing_tags = old_tag_names - new_tag_names
```

Comment:
How about calling it `removed_tags` so that it matches the parameters used in `automation.trigger!`? 🤔